### PR TITLE
Rename image/decoding symbols to follow Chromium style

### PR DESCRIPTION
**Pull request by bdero**
Rename things like `get_pixels` and camelcase methods. Avoided touching camelcase symbols with Dart bindings.

### CODE REVIEW

File: lib/ui/painting/image_descriptor.h [hunk 1]
```diff
@@ -26,7 +26,7 @@ namespace flutter {
 ///         This class will hold a reference on the underlying image data, and
 ///         in the case of compressed data, an `ImageGenerator` for the data.
 ///         The Codec initialization actually happens in initEncoded, making
-///         `initstantiateCodec` a lightweight operation.
+///         `InitstantiateCodec` a lightweight operation.
```
Comment (dnfield) [inline lib/ui/painting/image_descriptor.h:29]: Find and replace mistake?

**Review by dnfield — CHANGES_REQUESTED** [review]
Per https://google.github.io/styleguide/cppguide.html#Function_Names, accessors/mutators for object fields can be named like fields. We do this elsewhere. I think these are fine.

The `initEncoded` method is exposed to Dart with that casing, and it's ab it confusing to mix it - I think we should leave that one as well, or change _all_ of them for all the native peers instead of just this one.

**Comment by dnfield** [comment]
(It's telling, in particular, that many of these methods are defined inline in the header because they're just setting/getting a private ivar)

**Comment by bdero** [comment]
> Per https://google.github.io/styleguide/cppguide.html#Function_Names, accessors/mutators for object fields can be named like fields. We do this elsewhere. I think these are fine.

Ah, I had a suspicion I was missing something here.

> The initEncoded method is exposed to Dart with that casing, and it's ab it confusing to mix it - I think we should leave that one as well, or change all of them for all the native peers instead of just this one.

Agreed. At first I also just renamed all of the stuff bound to Dart, but realized that just shifts the style inconsistency to the otherwise pristine Dart that users will encounter way more often in practice and reverted it.

This resolved some of my lingering style confusion. Thanks!

**Pull request closed without merging**